lms/scripts/initdb.py

```diff
@@ -0,0 +1,14 @@
+"""
+A CLI command to initialize the database if it isn't already initialized.
+
+Usage:
+
+    initdb conf/development.ini
+"""
+import sys
+
+from pyramid.paster import bootstrap
+
+
+def initdb():
+    bootstrap(sys.argv[1])
```

Comment:
Initializing the DB (if it isn't already) is already done as part of booting up the app. For example `make dev` would do it. So all we need for a standalone script is to call `bootstrap` and then exit

Reply:
P.S. The way we've added a command line script to the app here is the same as how lms's `devdata` script is already done, and is just the standard way of doing command line scripts in Pyramid apps